README.md

```diff
@@ -120,6 +121,7 @@ _(scroll right on table to see all 5 columns)_
 | Instagram                      | [github.com (gif)](https://github.com/vipulasri/JetInstagram/blob/master/art/jetinstagram.gif)                                                                         | [GitHub](https://github.com/vipulasri/JetInstagram)                                                                                        | Jetpack Compose                                                                | ![GitHub Repo stars](https://flat.badgen.net/github/stars/vipulasri/JetInstagram)                                                                                                                |
 | Instagram                      | [instagram-clone-reactjs.vercel](https://instagramclone-react.vercel.app/)                                                                                             | [GitHub](https://github.com/mucahit-sahin/instagram-clone-react)                                                                           | React, Redux                                                                   | ![GitHub Repo stars](https://flat.badgen.net/github/stars/mucahit-sahin/instagram-clone-react)                                                                                                   |
 | Instagram                      | [instagram-clone-dbe40.web.app](https://instagram-clone-dbe40.web.app/)                                                                                                | [GitHub](https://github.com/shaan71845/instagram-clone)                                                                                    | React, Firebase                                                                | ![GitHub Repo stars](https://flat.badgen.net/github/stars/shaan71845/instagram-clone)                                                                                                            |
+| Instagram                      | [instagram-clone-redesign.vercel.app](https://github.com/jhony-v/instagram-clone-redesign)                                                                             | [GitHub](https://github.com/jhony-v/instagram-clone-redesign)                                                                              | React, Zustand, Styletron, NextJs, Javascript, GraphQL, Strapi                 | ![GitHub Repo stars](https://flat.badgen.net/github/stars/jhony-v/instagram-clone-redesign)                                                                                                            |
```

Comment:
any kind of video and images are not visible and auth. doesn't working .
Can't visit another one's profile .....